src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -27,12 +27,6 @@ bool StridedSlice::isSupportedOperation(const std::shared_ptr<const ov::Node>& o
                 !ov::is_type<ov::op::v8::Slice>(op)) {
             errorMessage = "Only StridedSlice from opset1 and Slice from opset8 operations are supported.";
             return false;
-
-            if (op->get_input_size() > AXES_ID && !ov::is_type<ov::op::v0::Constant>(op->get_input_node_ptr(AXES_ID))) {
-                // TODO: all required modifications are completed on the node level. More functional tests have to be implemented to resolve the limitation.
-                errorMessage = "Only constant 'axis' input is supported.";
-                return false;
-            }
```

Comment:
Why is this condition check removed? I was mostly a typo `{` symbol was missed.

Reply:
I had thought it was a typo, after adding this check CI ONNX test would fail. Then I review the strided_slice.cpp, It seems that AXIS can support runtime parameter.
https://github.com/openvinotoolkit/openvino/blob/f410658d32b369a144d85c2cbe4b08b669b17c9f/src/plugins/intel_cpu/src/nodes/strided_slice.cpp#L548

Reply:
Just checked with @dmitry-gorokhov , this code is still actual.

Reply:
@luweizhou2016 , could you please extend the existing strided slice SL scope with tests dedicated to the dynamic axis support, to satisfy the requirement?

Reply:
Sure. It would be safe to make the changes. Will update the PR once done.